demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -272,6 +273,36 @@ def validate_version_bump(self):
 
         return True
 
+    # Author_image.png validation
+    def validate_author_image_file(self, author_image_path) -> bool:
+        """
+        Validates Author_image.png.
+        For more info please visit https://xsoar.pan.dev/docs/packs/packs-format#author_imagepng
+        """
+        return all([self._is_author_image_not_empty(author_image_path),
+                    self._is_author_image_dimensions_valid(author_image_path),
+                    self._is_author_image_size_valid(author_image_path)])
+
+    def _is_author_image_not_empty(self, author_image_path) -> bool:
+        author_image_content = Image.open(author_image_path)
+        if not author_image_content:
+            self._add_error(Errors.author_image_fie_is_empty(), self.pack_meta_file)
+            return False
+        return True
+
+    def _is_author_image_dimensions_valid(self, author_image_path) -> bool:
+        author_image = Image.open(author_image_path)
+        if not author_image.size == AUTHOR_IMAGE_DIMENSIONS:
+            self._add_error(Errors.author_image_fie_invalid_dimensions(), self.pack_meta_file)
+            return False
+        return True
+
+    def _is_author_image_size_valid(self, author_image_path) -> bool:
+        if not os.path.getsize(author_image_path) <= AUTHOR_IMAGE_SIZE:
+            self._add_error(Errors.author_image_fie_invalid_size(), self.pack_meta_file)
+            return False
+        return True
```

Comment:
We shouldn't validate these as part of the init. We only validate these kind of thing in the validate command